### PR TITLE
Fixed grammatical mistakes and typos in Tutorial 6

### PR DESCRIPTION
**Pull request by gokulp01**
Fixed almost all of the grammatical mistakes and typos in the 4 Wheels Robot tutorial. It will make it easier for new learners to understand easily.

**Description**
There were a lot of grammatical mistakes and typos in Tutorial 6 that bottlenecks the understanding of the tutorial. 

**Tasks**
Add the list of tasks of this PR.
  - [ x ] Fix Grammatical mistakes
  - [  x ] Fix typos

**Documentation**
https://cyberbotics.com/doc/guide/tutorial-6-4-wheels-robot?tab-language=c++

**Additional context**
This would make it easier to read and understand the tutorial

### CODE REVIEW

**Comment by gokulp01** [comment]
@DavidMansolino @omichel  is there any particular reason why the tests are failing? I mean, all I did was make some changes to the markdown file.

**Comment by DavidMansolino** [comment]
@gokulp01 you can see the test output here:
https://travis-ci.com/github/cyberbotics/webots/builds/173331274#L248
The problem is that since you changed the caption of some figures, some links to them are not valid anymore, let me show you.

File: docs/guide/tutorial-6-4-wheels-robot.md [hunk 1]
```diff
@@ -68,13 +72,13 @@ The [figure](#high-level-representation-of-the-4-wheels-robot) depicts the solid
 Please refer to [this figure](#high-level-representation-of-the-4-wheels-robot).
 Add a [Shape](../reference/shape.md) node containing a Box geometry to the [Robot](../reference/robot.md) node.
 Set the color of the [Shape](../reference/shape.md) to red.
-Use the [Shape](../reference/shape.md) to define also the `boundingObject` field of the [Robot](../reference/robot.md) node.
+Use the [Shape](../reference/shape.md) to define the `boundingObject` field of the [Robot](../reference/robot.md) node.
 The dimension of the box is `(0.1, 0.05, 0.2)`.
 Add a [Physics](../reference/physics.md) node to the [Robot](../reference/robot.md).
 The [figure](#low-level-representation-of-the-4-wheels-robot) represents all the nodes defining the robot.
```
Comment (DavidMansolino) [inline docs/guide/tutorial-6-4-wheels-robot.md:78]:
```suggestion
The [figure](high-level-representation-of-the-4-wheeled-robot) represents all the nodes defining the robot.
```

File: docs/guide/tutorial-6-4-wheels-robot.md [hunk 2]
```diff
@@ -1,65 +1,69 @@
...
-The [next figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices) shows the robot from a top view.
+This tutorial aims at creating your first robot from scratch.
+The robot will consist of a body, four wheels, and two distance sensors.
+The result is depicted in [this figure](#3d-view-of-the-4-wheels-robot-note-that-the-coordinate-system-representations-of-the-robot-body-and-of-its-wheels-are-oriented-the-same-way-their-px-vector-defines-the-left-of-the-robot-their-py-vector-defines-the-top-of-the-robot-and-their-pz-vector-defines-the-front-of-the-robot-the-distance-sensors-are-oriented-in-a-different-way-their-px-vector-indicates-the-direction-of-the-sensor).
```
Comment (DavidMansolino) [inline docs/guide/tutorial-6-4-wheels-robot.md:5]:
```suggestion
The result is depicted in [this figure](3d-view-representation-of-the-4-wheeled-robot-note-that-the-coordinate-system-representations-of-the-robot-body-and-its-wheels-are-oriented-the-same-way-the-px-vector-in-red-defines-the-left-of-the-robot-the-y-vector-in-green-defines-the-top-of-the-robot-and-their-z-vector-in-blue-defines-the-front-of-the-robot-the-distance-sensors-are-oriented-differently-their-x-vector-indicates-the-direction-of-the-sensor).
```

File: docs/guide/tutorial-6-4-wheels-robot.md [hunk 3]
```diff
@@ -1,65 +1,69 @@
...
+The [next figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices) shows the top view of the robot.
```
Comment (DavidMansolino) [inline docs/guide/tutorial-6-4-wheels-robot.md:6]:
```suggestion
The [next figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-device) shows the top view of the robot.
```

File: docs/guide/tutorial-6-4-wheels-robot.md [hunk 4]
```diff
@@ -68,13 +72,13 @@ The [figure](#high-level-representation-of-the-4-wheels-robot) depicts the solid
 Please refer to [this figure](#high-level-representation-of-the-4-wheels-robot).
```
Comment (DavidMansolino) [inline docs/guide/tutorial-6-4-wheels-robot.md:72]:
```suggestion
Please refer to [this figure](#high-level-representation-of-the-4-wheeled-robot).

File: docs/guide/tutorial-6-4-wheels-robot.md [hunk 5]
```diff
@@ -123,13 +128,13 @@ While the rotation origin (anchor) and the rotation axis (axis) are defined by t
...
+Finally, the [HingeJointParameters](../reference/hingejointparameters.md) axis should define the rotation axis.
 In our case it's along the x-axis (so `(1, 0, 0)`).
 
 > **Hands on #3**: Add a [HingeJointParameters](../reference/hingejointparameters.md) node, and enter the field values as described above.
 Some signs obviously have to be updated for other wheels.
 
-We want now to implement the cylinder shape of the wheels.
+Now, let's implement the cylinder shape of the wheels.
 
 > **Hands on #4**: Complete the missing nodes to get the same structure as the one depicted in [this figure](#low-level-representation-of-the-4-wheels-robot).
```
Comment (DavidMansolino) [inline docs/guide/tutorial-6-4-wheels-robot.md:139]:
```suggestion
> **Hands on #4**: Complete the missing nodes to get the same structure as the one depicted in [this figure](#low-level-representation-of-the-4-wheeled-robot).
```

File: docs/guide/tutorial-6-4-wheels-robot.md [hunk 6]
```diff
@@ -143,14 +148,14 @@ Set the color of the wheels to green.
...
+The sensors used on the e-puck of [Tutorial 4](tutorial-4-more-about-controllers.md) are not the same as the ones used in this section. Pay attention to their measuring interval which is between `0 cm = 0` and `10 cm = 1000`.
 Find more information about the `lookupTable` field [here](../reference/distancesensor.md#lookup-table).
 
 
 The last part of the robot modeling is to add the two distance sensors to the robot.
 This can be done by adding two [DistanceSensor](../reference/distancesensor.md) nodes as direct children of the [Robot](../reference/robot.md) node.
 Note that the distance sensor acquires its data along the +*x*-axis.
-So rotating the distance sensors in order to point their *x*-axis outside the robot is necessary (see the [figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices)).
+So, it is necessary to rotate the distance sensor to point their x-axis outside the robot (see the [figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices)).
```
Comment (DavidMansolino) [inline docs/guide/tutorial-6-4-wheels-robot.md:158]:
```suggestion
So, it is necessary to rotate the distance sensor to point their x-axis outside the robot (see the [figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-device)).
```

File: docs/guide/tutorial-6-4-wheels-robot.md [hunk 7]
```diff
@@ -159,10 +164,10 @@ As already says in [Tutorial 3](tutorial-3-appearance.md), the distance sensor r
...
 
 > **Hands on #5**: Add the two distance sensors as explained above.
-The distance sensors are at an angle to 0.3 [rad] with the robot front vector.
-Set their graphical and physical shape to a cube (not transformed) having a edge of `0.01` [m].
-Set their color to blue.
-Set their `name` field according to the labels of [this figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices).
+The distance sensors are at an angle to 0.3 [rad] with the front vector of the robot.
+Set thei graphical and physical shape to a cube (not transformed) having a edge of `0.01` [m].
+Set thei color to blue.
+Set thei `name` field according to the labels of [this figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices).
```
Comment (DavidMansolino) [inline docs/guide/tutorial-6-4-wheels-robot.md:170]:
```suggestion
Set thei `name` field according to the labels of [this figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-device).
```

Comment (gokulp01) [reply docs/guide/tutorial-6-4-wheels-robot.md:170]: Oh yes, I see my mistake here. Thanks a lot for pointing them out :)

Comment (DavidMansolino) [reply docs/guide/tutorial-6-4-wheels-robot.md]: You're welcome, thank you for your help making the doc better.

**Comment by gokulp01** [comment]
@DavidMansolino  Thank you for pointing out my mistakes. Some tests are still failing. I will fix them and commit the changes. Thanks again :)

File: docs/guide/tutorial-6-4-wheels-robot.md [hunk 8]
```diff
@@ -1,80 +1,84 @@
...
 The root node of this tree should be a [Robot](../reference/robot.md) node.
-The solids are linked together by [Joint](../reference/joint.md) nodes.
+
+- The solids are linked together by [Joint](../reference/joint.md) nodes.
 A [Device](../reference/device.md) node should be the direct child of either a [Robot](../reference/robot.md) node, a [Solid](../reference/solid.md) node or a [Joint](../reference/joint.md) node.
 
-A [Joint](../reference/joint.md) node is used to add one (or two) degree(s) of freedom (DOF), between its parent and its child.
-The direct parent and child of a [Joint](../reference/joint.md) node are both [Solid](../reference/solid.md) nodes.
+- A [Joint](../reference/joint.md) node is used to add one (or two) degree(s) of freedom (DOF), between its parent and its child.
+The direct parent and child of a [Joint](../reference/joint.md) nodes are both [Solid](../reference/solid.md) nodes.
```
Comment (omichel) [inline docs/guide/tutorial-6-4-wheels-robot.md:44]: I believe the singular form was correct here.
```suggestion
The direct parent and child of a [Joint](../reference/joint.md) node are both [Solid](../reference/solid.md) nodes.
```

File: docs/guide/tutorial-6-4-wheels-robot.md [hunk 9]
```diff
@@ -159,10 +164,10 @@ As already says in [Tutorial 3](tutorial-3-appearance.md), the distance sensor r
...
 
 > **Hands on #5**: Add the two distance sensors as explained above.
-The distance sensors are at an angle to 0.3 [rad] with the robot front vector.
-Set their graphical and physical shape to a cube (not transformed) having a edge of `0.01` [m].
-Set their color to blue.
-Set their `name` field according to the labels of [this figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices).
+The distance sensors are at an angle to 0.3 [rad] with the front vector of the robot.
+Set thei graphical and physical shape to a cube (not transformed) having a edge of `0.01` [m].
+Set thei color to blue.
+Set thei `name` field according to the labels of [this figure](#top-view-of-the-4-wheeled-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-device).
```
Comment (omichel) [inline docs/guide/tutorial-6-4-wheels-robot.md:170]: "thei" seems wrong, should be "the" I guess?

Comment (gokulp01) [reply docs/guide/tutorial-6-4-wheels-robot.md:170]: oh yes, my bad

**Comment by gokulp01** [comment]
@DavidMansolino @omichel  Thanks a lot. May I close this PR ?

**Pull request closed without merging**